Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:10
```

Comment:
Not sure whether this should live at top-level or under server dir, it's ok here for now.